fava/core/__init__.py

```diff
@@ -355,6 +355,47 @@ def context(self, entry_hash):
         source_slice, sha256sum = get_entry_slice(entry)
         return entry, balances, source_slice, sha256sum
 
+    def render_journal(self):
```

Comment:
There's no need for another function on FavaLedger. This can be more cleanly done with a template.

Reply:
Changed this to a template in 48c3239